openbas-api/src/main/java/io/openbas/rest/helper/RestBehavior.java

```diff
@@ -92,7 +93,7 @@ public ValidationErrorBag handleBadRequestExceptions(ImportException ex) {
   }
 
   @ResponseStatus(HttpStatus.UNAUTHORIZED)
-  @ExceptionHandler(AccessDeniedException.class)
+  @ExceptionHandler(AuthenticationException.class)
   public ValidationErrorBag handleValidationExceptions() {
     ValidationErrorBag bag =
         new ValidationErrorBag(HttpStatus.UNAUTHORIZED.value(), "ACCESS_DENIED");
```

Comment:
Maybe change default message for something more meaningful as UNATHORIZED is described as : "it lacks valid authentication credentials for the requested resource"